apps/sso/views.py

```diff
@@ -1,18 +1,9 @@
-# -*- encoding: utf-8 -*-
-
 import logging
 
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
 from oauth2_provider.views.base import AuthorizationView as DefaultAuthorizationView
 
-# fmt: off
-from oauth2_provider.views.base import (  # noqa: F401  # isort:skip
-        RevokeTokenView,
-        TokenView,
-)
-# fmt: on
-
```

Comment:
Turns out this was only used for re-exporting https://github.com/dotkom/onlineweb4/commit/5c91652ac28f074febd6cded9654d095d32145ce#diff-fbf840138d1fbdea4b2c085b1cd0288c